src/main/java/net/earthcomputer/clientcommands/command/CTimeCommand.java

```diff
@@ -1,13 +1,19 @@
 package net.earthcomputer.clientcommands.command;
 
 import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.arguments.IntegerArgumentType;
```

Comment:
This import is unused since the `IntegerArgumentType.getInteger` method is imported statically instead.